tasks.yaml

```diff
@@ -0,0 +1,81 @@
+tasks:
+  - name: set-modules-variable
+    actions:
+      - cmd: echo $(find . -type f -name 'go.mod' -exec dirname {} \; | cut -c 3-)
+        setVariables:
+          - name: MODULES
+
+  # This could be improved if something like this was introduced https://github.com/defenseunicorns/maru-runner/issues/17
```

Comment:
use `--with`, not `--set` to accomplish this, source: me

Reply:
deleting task file for now based on discussion